gensim/test/test_doc2vec.py

```diff
@@ -123,6 +123,18 @@ def test_empty_errors(self):
         # input not empty, but rather completely filtered out
         self.assertRaises(RuntimeError, doc2vec.Doc2Vec, list_corpus, min_count=10000)
 
+    def test_oot_similarity(self):
+        """Test similarity of out of training sentences"""
+        empty_str = ''
+        rome_str = 'rome italy'
+        car_str = 'car'
+        corpus = list(DocsLeeCorpus(True))
+
+        model = doc2vec.Doc2Vec(min_count=1)
+        model.build_vocab(corpus)
+        self.assertEqual(int(model.docvecs.oot_similarity(model, '', '')), 1)
+        self.assertTrue(model.docvecs.oot_similarity(model, rome_str, rome_str) > model.docvecs.oot_similarity(model, rome_str, car_str))
```

Comment:
This also is not likely testing what's expected, as strings are being passed rather than lists-of-tokens. I'd guess 'rome italy' is being interpreted as ['r', 'o', 'm', 'e', ' ', 'i', 't', 'a', 'l', 'y'], which becomes something like just ['i', 'a'] when reduced to known-tokens. Then 'car' becomes ['a'] via the same process.